src/pages/Authentication/components/Input/components/Textarea/Textarea.tsx

```diff
@@ -158,16 +70,24 @@ export const Textarea = (props: TextareaPropsType) => {
       <Field
         component='textarea'
         name='token'
-        onInput={onInput}
-        onChange={onChange}
+        onInput={handleInput}
+        onChange={handleChange}
         className={classNames(styles.field, {
           [styles.large]: Boolean(nativeAuthToken)
         })}
       />
 
-      <div className={styles.clone}>
-        <div className={styles.mirror} ref={clone}>
-          {mirror.map((word: any, index: any) => (
+      <div
+        className={styles.clone}
+        ref={clone}
+        contentEditable='true'
```

Comment:
Without true?

Reply:
it's mandatory, you can set only 2 values here: "true" or "false" as string